core/commons/che-core-commons-inject/src/main/java/org/eclipse/che/inject/CheBootstrap.java

```diff
@@ -438,18 +438,22 @@ private void bindProperty(String prefix, String name, String value) {
       String key = prefix == null ? name : (prefix + name);
       Set<String> aliasesForName = bindMap.get(name);
       if (value == null) {
+        LOG.debug("Binding `{}` to `null`", key);
         bind(String.class).annotatedWith(Names.named(key)).toProvider(Providers.<String>of(null));
         if (aliasesForName != null) {
           for (String alias : aliasesForName) {
+            LOG.debug("Binding `{}` to `null`", alias);
             bind(String.class)
                 .annotatedWith(Names.named(prefix == null ? alias : prefix + alias))
                 .toProvider(Providers.<String>of(null));
           }
         }
       } else {
+        LOG.debug("Binding `{}` to `{}`", key, value);
         bindConstant().annotatedWith(Names.named(key)).to(value);
         if (aliasesForName != null) {
           for (String alias : aliasesForName) {
+            LOG.debug("Binding `{}` to `{}`", key, value);
```

Comment:
This is wrong, I think? We're binding the aliases here, not the key itself. Also lookout for the `prefix` value.